### PR TITLE
update dependency

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubeedge/website/issues/381

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @kevin-wangzefeng

**Comment by Shelley-BaoYue** [comment]
I think it's better to use the latest patch version.  5.7.3 or 5.8.4 is more suitable for antd.

File: package.json [hunk 1]
```diff
@@ -18,18 +18,18 @@
...
     "react-dom": "^17.0.2",
     "react-slick": "^0.29.0",
-    "sass": "^1.63.6",
+    "sass": "^1.66.0",
     "slick-carousel": "^1.8.1"
   },
   "devDependencies": {
     "@docusaurus/module-type-aliases": "2.4.1",
-    "antd": "^5.7.0"
+    "antd": "^5.8.3"
```
Comment (Shelley-BaoYue) [inline package.json:32]: It's better to use the latest patch version. 5.7.3 or 5.8.4 is more suitable.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**